ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -129,10 +133,47 @@ Test Inference With Token Authentication
 
 Verify Multiple Projects With Same Model
     [Documentation]    Test the deployment of multiple DS project with same openvino_ir model
-    [Tags]    Sanity
-    ...    RHOAIENG-549    RHOAIENG-2724
+    [Tags]    Sanity    Tier1
+    ...       RHOAIENG-549    RHOAIENG-2724
     Create Openvino Models    server_name=${RUNTIME_NAME}    model_name=${MODEL_NAME}    project_name=${PRJ_TITLE}
-    ...    num_projects=5
+    ...    num_projects=4
+    [Teardown]    Run Keyword If Test Failed    Get Modelmesh Events And Logs
+    ...    server_name=${RUNTIME_NAME}    project_title=${PRJ_TITLE}
+
+Verify Editing Existing Model Deployment
+    [Documentation]    Tries editing an existing model deployment to see if the underlying deployment is updated
+    [Tags]    Sanity    Tier1
+    ...       ProductBug    RHOAIENG-2869
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Recreate S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=model-serving-connection
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=ods-ci-s3
+    Create Model Server    token=${FALSE}    server_name=${RUNTIME_NAME}    existing_server=${TRUE}
+    Serve Model    project_name=${PRJ_TITLE}    model_name=${MODEL_NAME}    framework=tensorflow
+    ...    existing_data_connection=${TRUE}    data_connection_name=model-serving-connection
+    ...    model_path=inception_resnet_v2.pb
+    ${runtime_pod_name}=    Replace String Using Regexp    string=${RUNTIME_NAME}    pattern=\\s    replace_with=-
+    ${runtime_pod_name}=    Convert To Lower Case    ${runtime_pod_name}
+    Wait Until Keyword Succeeds    5 min  10 sec  Verify Openvino Deployment    runtime_name=${RUNTIME_POD_NAME}
+    Wait Until Keyword Succeeds    5 min  10 sec  Verify Serving Service
+    Verify Model Status    ${MODEL_NAME}    success
+    Set Suite Variable    ${MODEL_CREATED}    ${TRUE}
+    ${url}    ${kserve}=    Get Model Route via UI    ${MODEL_NAME}
+    ${status_code}    ${response_text}=    Send Random Inference Request     endpoint=${url}    name=input
+    ...    shape={"B": 1, "H": 299, "W": 299, "C": 3}    no_requests=1
+    Should Be Equal As Strings    ${status_code}    200
+    Serve Model    project_name=${PRJ_TITLE}    model_name=${MODEL_NAME}    framework=openvino_ir
+    ...    existing_data_connection=${TRUE}    data_connection_name=model-serving-connection
+    ...    model_path=openvino-example-model    existing_model=${TRUE}
```

Comment:
I wonder if it would be more clear to have a keyword `Edit Served Model`

Reply:
I find it less readable and predictable

Reply:
Sticking with this format for now, we can discuss tomorrow if we want to change it everywhere